util/dvsim/Launcher.py

```diff
@@ -60,6 +61,9 @@ class Launcher:
     # more than this many directories.
     max_odirs = 5
 
+    # Record job runtime.
+    job_runtime = 0
```

Comment:
IMO this is a class instance property, not a global property. It must be initialized in `__init__` as `self.job_runtime = 0`. 
I am a little confused - why duplicate `job_runtime` in `Deploy` as well? I think it would be better to only maintain at one place to avoid confusion.

Reply:
Thanks Sri! I understand your point for creating a property instead of he global property. Thanks for explaining that.
As for duplciate `job_runtime` in `Deploy` - In `Deploy`, we create different class for different target (run, build, cov etc). So I thought maintain a different copy for each category would be useful to generate result.

Reply:
Right - all `Deploy` class derivatives internally create a private localized launcher class instance. So a deploy object (whatever it is) can reference `job_runtime` via the launcher class hierarchically as `deploy_obj.launcher.job_runtime`. The launcher obj also holds a handle to its parent deploy class via the `self.deploy` field, so really, only need to maintain `job_runtime` as launcher class instance property.